extensions/2.0/Vendor/EXT_feature_metadata/schema/primitive.EXT_feature_metadata.featureMapping.featureIds.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Per-vertex feature IDs",
+    "type": "object",
+    "description": "An object describing per-vertex feature IDs to be used as indices to property arrays in the feature collection.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "attribute": {
+            "type": "string",
+            "pattern": "^_FEATURE_ID_([1-9]\\d*|0)$",
+            "description": "The name of the attribute containing feature IDs. Feature IDs must be whole numbers in the range `[0, elementCount - 1]`, where `elementCount` is the total number of elements in each property array."
+        },
+        "constant": {
+            "type": "integer",
+            "minimum": 0,
+            "default": 0,
+            "description": "Sets a default feature ID for each vertex when the attribute property is not supplied. TODO: better name needed."
+        },
+        "instanceStride": {
```

Comment:
I'm still feel unsure that these strides will be as helpful as the `constant` above, this should get further discussion in time.

Reply:
We should rethink instancing in general. Pretty sure this will just confuse people.